examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -6,7 +6,7 @@
 import { Server } from "http";
 
 import cors from "cors";
-import express from "express";
+import express, { Request } from "express";
```

Comment:
Super nit: let's just use `express.Request` below, so we don't mix the import line?